agent/php_error.c

```diff
@@ -466,30 +477,51 @@ static char* nr_php_error_exception_message(zval* exception TSRMLS_DC) {
   return message;
 }
 
-static const char* get_error_type_string(int type) {
+const char* nr_get_error_type_string(int type) {
```

Comment:
The name of this function seems too generic. There's not enough context. Usually the convention for exported functions is to prefix them with:
- `nr_` to indicate it is New Relic space
- <filename> to indicate where the code for the function is; this gives the context of what a given function is about

Therefore I suggest updating it to `nr_php_error_get_type_string`.

Reply:
Please see: [188cf39](https://github.com/newrelic/newrelic-php-agent/pull/850/commits/188cf391b2cc618dc4346a42ffc44a00d7967f1b)